### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by anandasaia** [HassanBinHaroon/YOLOER_V2#2]
Is there any documentation or further details regarding the models and algorithms used available?

**Comment by HassanBinHaroon** [comment]
> Is there any documentation or further details regarding the models and algorithms used available?

"YOLOER stands for You Only Look Once and Estimate Range while V1 implies the version of object detector (V2 = YOLOv7). The project is about REAL-TIME Object Detection and Distance/Depth Estimation using YOLOv7 object detector. To test the Real-Time functionality immediately, we provide the option of running the model on the webcam. Moreover, the options of testing on recorded videos and images are also provided."

**Comment by HassanBinHaroon** [comment]
> Is there any documentation or further details regarding the models and algorithms used available?

[Details](https://github.com/HassanBinHaroon/YOLOER_V2/issues/1)

**Comment by anandasaia** [comment]
I meant documentation in the sense that which algorithm is being used for the distance estimation? What kind of neural network is being used? Is it a perceptron network or a CNN, things like that?

**Comment by HassanBinHaroon** [comment]
A perceptron network.

**Comment by abdulghani91** [comment]
@HassanBinHaroon 

Hi, thank you for this fantastic project.

Do you have any ideas about the inputs for the perceptron network, or is there a specific source for the training dataset?

**Comment by HassanBinHaroon** [comment]
If you want to discuss, we can have an online session. If interested, then
share your email.

**Comment by abdulghani91** [comment]
@HassanBinHaroon 

Yes, sure, I am interested. Here is my email: abdulghani.alrawi91@gmail.com
